cmd/kro/go.mod

```diff
@@ -0,0 +1,79 @@
+module github.com/kro-run/kro/cmd/kro
+
+go 1.24.0
+
+require (
+	github.com/go-echarts/go-echarts/v2 v2.6.5
+	github.com/kubernetes-sigs/kro v0.5.0
```

Comment:
How do we plan to handle version bumps?
in other words, how do we make sure that the CLI uses the same version, in particular for development and `latest` builds.

In my experience, you need to have the version pushed to be able to do so, which is a bit annoying when developing (for example updating the CRD definition or the emulator itself)